src/util/helpers.ts

```diff
@@ -75,9 +75,9 @@ export const toBitcoinAmount = (amountInSatoshi: string): string => {
   let value;
 
   if (bnAmountInSatoshi.lt(bnSatoshis)) {
-    value = parseInt(amountInSatoshi, 10) / SATOSHIS;
-  } else {
     value = bnAmountInSatoshi.div(bnSatoshis).toNumber(); // Okay to do toNumber as we're dealing with TKL and not satoshis
+  } else {
+    value = parseInt(amountInSatoshi, 10) / SATOSHIS;
```

Comment:
Haven't tested this one yet, but theoretically this inverted logic doesn't make sense imo.

The original conditional states: 
```
  if the transaction amount is less than 1 TKL (100,000,000 satoshis)
  then
    do a normal JS math division, i.e. the amount of the transaction divided by 100,000,000 (`SATOSHIS`)
  else
    do the same mathematical operation but using [BN.js](https://github.com/indutny/bn.js) to be sure we don't go over `MAX_SAFE_INTEGER`, and convert the result to a normal js number
```

We need this conditional because BN.js doesn't support decimals or floats, so if our division results in <1, it kinda breaks. That's why if the number we're dividing is less than the amount of satoshis in a unit of TKL, we use normal JS. This change seems to invert this, so I'm 🤔🤔🤔🤔🤔🤔